src/ML-IAP/pair_mliap.cpp

```diff
@@ -131,52 +131,53 @@ void PairMLIAP::settings(int narg, char ** arg)
 {
   if (narg < 2) utils::missing_cmd_args(FLERR, "pair_style mliap", error);
 
-  // set flags for required keywords
-
-  delete model;
-  model = nullptr;
-  delete descriptor;
-  descriptor = nullptr;
-
   // process keywords
 
   int iarg = 0;
 
+  //Check to see if there are more than one model or descriptor
```

Comment:
This looks very "hack-ish" and I am not convinced that this is a good approach. See my comment for the ::settings() function in the KOKKOS version.